toolchain/driver/lld_runner.cpp

```diff
@@ -0,0 +1,67 @@
+// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+// Exceptions. See /LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include "toolchain/driver/lld_runner.h"
+
+#include <algorithm>
+#include <memory>
+#include <numeric>
+#include <optional>
+
+#include "common/vlog.h"
+#include "lld/Common/Driver.h"
+#include "llvm/ADT/ArrayRef.h"
+#include "llvm/ADT/StringRef.h"
+
+// Declare the supported driver flavor entry points.
+// NOLINTNEXTLINE(readability-identifier-naming): External library name.
+LLD_HAS_DRIVER(elf)
+// NOLINTNEXTLINE(readability-identifier-naming): External library name.
+LLD_HAS_DRIVER(macho)
+
+namespace Carbon {
+
+LLDRunner::LLDRunner(const InstallPaths* install_paths,
+                     llvm::raw_ostream* vlog_stream)
+    : ToolRunnerBase(install_paths, vlog_stream) {}
```

Comment:
Could this be replaced with `using ToolRunnerBase::ToolRunnerBase;` in the header?